src/inausoft.netCLI.Tests/LogicalCommandDeserializerTests.cs

```diff
@@ -39,11 +38,56 @@ public void RegexArgumentHandler_DeserializeProperCommand_ForMultipleOptions()
             Assert.AreEqual(intOptionValue, command.IntOption);
         }
 
+        [TestMethod]
+        public void LogicalCommandDeserializer_DeserializeProperCommand_ForMultipleOptions_UsingShortNames()
+        {
+            //Arrange
```

Comment:
As code refactor we should correct all tests to proper formatting:
```suggestion
            // Arrange
```

Reply:
Agreed.